src/sdk/object_sdk.js

```diff
@@ -180,7 +180,9 @@ class ObjectSDK {
         try {
             // NAMESPACE_FS HACK
             if (process.env.NAMESPACE_FS) {
-                const namespace_resource_id = bucket.namespace.write_resource?.resource.id;
+                const namespace_resource_id =
+                    bucket.namespace.write_resource &&
+                    bucket.namespace.write_resource.resource.id;
```

Comment:
maybe 
`const namespace_resource_id = _.extend({}, bucket.namespace.write_resource).resource.id;`
WDYT?

Reply:
I prefer a straightforward code instead of trying to understand why extending an empty object with this other object is meant to do...

Reply:
👍